Sources/Locker/Helpers/LockerHelpers.swift

```diff
@@ -91,7 +87,7 @@ class LockerHelpers {
             // In case lib is used on simulator, error code will always be `notEnrolled` and only then
             // we want to return that biometrics is not supported as we don't know what simulator is used.
             if let error = error,
-               error.code == biometryNotAvailableCode || (error.code == biometryNotEnrolledCode && isSimulator) {
```

Comment:
I would check this one, as I think we had an issue with false info on older simulators.

If the test shows that this is now resolved and it's working, please remove the comment above. In that case, it's not correct anymore.